lib/Backend/IRBuilder.cpp

```diff
@@ -3296,6 +3320,12 @@ IRBuilder::BuildElementSlot(Js::OpCode newOpcode, uint32 offset, Js::RegSlot fie
     StackSym *      stackFuncPtrSym = nullptr;
     bool isLdSlotThatWasNotProfiled = false;
 
+    IR::RegOpnd *   src0Opnd;
```

Comment:
If you put the body of the case in a `{`block`}` then you can declare these variables there and it will compile happily.